docs/examples/azure/gitversion/execute.md

```diff
@@ -120,9 +120,9 @@ The Execute GitVersion task creates the following job-scoped variables and multi
 
 ```yaml
 steps:
-  # gitversion/setup@3.2.1 task omitted for brevity.
+  # gitversion/setup@3.3.0 task omitted for brevity.
```

Comment:
please revert all the version changes, the version gets updated by the release pipeline

Reply:
Ah, good to know. Sorry about that